Sources/ApolloCodegenLib/FileGenerators/InputObjectFileGenerator.swift

```diff
@@ -7,8 +7,9 @@ struct InputObjectFileGenerator: FileGenerator, Equatable {
   let path: String
 
   var data: Data? {
-    #warning("TODO: need correct data template")
-    return "public struct \(inputObjectType.name) {}".data(using: .utf8)
+    InputObjectTemplate(graphqlInputObject: inputObjectType)
+      .render()
+      .data(using: .utf8)
```

Comment:
We should probably be wary of how this affects things for character-based languages like Japanese and Chinese which take heavy advantages of UTF-16. Not sure exactly how to test it but definitely worth at least making an issue on it

Reply:
That's an interesting point. I created https://github.com/apollographql/apollo-ios/issues/2115 so we follow up before 1.0 GA release.